src/main/java/co/rsk/bitcoinj/core/PartialMerkleTree.java

```diff
@@ -244,8 +244,15 @@ private void checkNotAValid64ByteTransaction(byte[] left, byte[] right) throws V
 
         // Skip input 0 outpoint hash
         _offset += 32;
-        // Skip input 0 outpoint index
+
+        // Check output 0 index
+        long output0Index = Utils.readUint32(leftAndRight, _offset);
```

Comment:
This looks good because it respects the `TransactionOutPoint` format:
https://github.com/rsksmart/bitcoinj-thin/blob/cfc84d827264262c05028f714e6acc4a60a7074d/src/main/java/co/rsk/bitcoinj/core/TransactionOutPoint.java#L94-L98